crowdsec-docs/docs/configuration/crowdsec_configuration.md

```diff
@@ -646,6 +740,54 @@ Path to certificate file.
 
 Path to certficate key file.
 
+###### `client_verification`
+
+Whether LAPI should require or not a client certificate for authentication.
+
+Supported values mirror the ones available in the [golang TLS library](https://pkg.go.dev/crypto/tls#ClientAuthType).
+
+Default to `VerifyClientCertIfGiven` which will allow connection without certificate or require a valid client certificate if one is provided
+
+::: warning
```

Comment:
I think the space makes it not render correctly :

![image](https://user-images.githubusercontent.com/990714/175944882-803b499a-e5a2-4e6f-86c6-548033774fd7.png)